docs/grafeas_concepts.md

```diff
@@ -86,3 +86,4 @@ the type of information each kind of note and occurrence contains.
 |IMAGE        |Information about the base image of a container|Information about layers included on top of the base image in a particular container|
 |PACKAGE      |Package descriptions                           |Filesystem locations detailing where the package is installed in a specific resource|
 |VULNERABILITY|CVE or vulnerability description and details including severity, versions|Affected packages/versions in a specific resource|
+|INTOTO|An in-toto step|Details of a particular in-toto link attestation of the corresponding step|
```

Comment:
Great! Let's also add a link to in-toto documentation here?

Reply:
Added a link to the specification.

Reply:
Great, thank you!